file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,588 @@
+  TYPES:
+    "! <p class="shorttext">Assign Event</p>
+    "! Assign event to consistency scenario
+    BEGIN OF ty_event,
+      "! <p class="shorttext">Event Name</p>
+      "! Event name
+      event               TYPE ty_event_name,
+      "! <p class="shorttext">Assign Checks and Comparison Type</p>
+      "! Assign Checks and Comparison Type
+      assigned_comparison TYPE ty_assigned_comparisons,
+      "! <p class="shorttext">Check Relevance</p>
+      "! Define relevance of checks
+      check_relevance     TYPE ty_check_relevances,
+    END OF ty_event.
+  "! <p class="shorttext">Assign Events</p>
+  "! Assign events to consistency scenario
+  TYPES ty_events TYPE SORTED TABLE OF ty_event WITH UNIQUE KEY event.
+  TYPES:
+    "! <p class="shorttext">Representation Type</p>
+    "! Assign representation type associated with consistency scenario
+    BEGIN OF ty_relationship_attribute,
+      "! <p class="shorttext">Representation Type</p>
+      "! Representation type
+      representation_type TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_relationship_attribute.
+  "! <p class="shorttext">Representation Types</p>
+  "! Assign representation types associated with consistency scenario
+  TYPES ty_relationship_attributes TYPE SORTED TABLE OF ty_relationship_attribute WITH UNIQUE KEY representation_type.
+  TYPES:
+    "! <p class="shorttext">eDocument Type Assignment</p>
+    "! Assign eDocument types associated with consistency scenario
+    BEGIN OF ty_edoc_type,
+      "! <p class="shorttext">eDocument Type</p>
+      "! eDocument Type
+      edoc_type TYPE c LENGTH 10,
+    END OF ty_edoc_type.
+  "! <p class="shorttext">eDocument Type Assignment</p>
+  "! Assign eDocument types associated with consistency scenario
+  TYPES ty_edoc_types  TYPE SORTED TABLE OF ty_edoc_type WITH UNIQUE KEY edoc_type.
+  "! <p class="shorttext">Inconsistency Category</p>
+  "! Inconsistency category
+  TYPES ty_resultgroup TYPE c LENGTH 20.
+  TYPES:
+    "! <p class="shorttext">Result Process</p>
+    "! Assign result process to the UI group
+    BEGIN OF ty_results_process,
+      result_process TYPE ty_result_process,
+    END OF ty_results_process.
+  "! <p class="shorttext">Result Processes</p>
+  "! Assign result process to the UI group
+  TYPES ty_result_processes TYPE SORTED TABLE OF ty_results_process WITH UNIQUE KEY result_process.
+  TYPES:
+    "! <p class="shorttext">Inconsistency Category</p>
+    "! Inconsistency category
+    BEGIN OF ty_inconsistency_category,
+      "! <p class="shorttext">Inconsistency Category</p>
+      "! Inconsistency category
+      result_ui_group  TYPE ty_resultgroup,
+      "! <p class="shorttext">Country View Extension</p>
+      "! Country view extension
+      country_xtension TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Result Processes</p>
+      "! Assign result process to the UI group
+      result_process   TYPE ty_result_processes,
```

Comment:
These result processes have been there before (field of checks). Do we need them twice?